src/main/java/frc/robot/Robot.java

```diff
@@ -33,12 +39,39 @@ public void robotInit() {
 
   @Override
   public void configureButtonBindings() {
-    driveController.back().whenPressed(drive.resetCmd());
 
-    DoubleSupplier trigger = driveController::getTriggers;
+    // list of buttons used by driver... back, x, a, b, UP, DOWN, START
+
+    // avalable buttons for driver controller... y, right bumper, left bumper,
+    // left stick, right stick
+
+    // list of button used by copilot... a, x, left bumper, right bumper
+
+    // avalable button for copilot controller... UP, DOWN, LEFT, RIGHT, y, b,
+    // start, back, left stick, right stick
+
+    // shooter bindings
+
+    copilotController.getPovButton(POVDirection.UP).whenPressed(shooter.setDefaultSpeed(DefaultSpeed.HIGH));
+    // sets upper goal speed
+    copilotController.getPovButton(POVDirection.DOWN).whenPressed(shooter.setDefaultSpeed(DefaultSpeed.LOW));
+    // sets lower goal speed
+    copilotController.lbumper().whileHeld(shooter.setSpeed(copilotController::getLeftTriggerAxis));
+    // when left bumper is pressed lets pilot set speed :)
+    copilotController.rbumper().whenPressed(sequence("Shoot",
```

Comment:
If this whole sequence is shooter-specific, can it be created in the shooter subsystem?